app/queries/crash.ts

```diff
@@ -223,7 +213,7 @@ export const UPDATE_CRASH = gql`
     update_crashes(where: { id: { _eq: $id } }, _set: $updates) {
       affected_rows
       returning {
-        cris_crash_id
+        id
```

Comment:
this is a good place to point out that we still have this awkward hard-coding of columns that should be returned by this graphql query, and we also need to make sure these columns are defined in the zod schema, and we also need to define these fields in the `crashesColumns.tsx` config. 

for the `crashesListView`, we work around this because the `queryBuilder` generates the graphql query from the config object, which itself is pulling it's columns from the `crashesListViewColumns` config file.